### PR TITLE
Adding shell build file

### PR DESCRIPTION
**Pull request by francescolops**

### CODE REVIEW

File: scripts/build/macos/start-client.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cd ../../../bin/
+java -jar clientCM.jar
```
Comment (adellafrattina) [inline scripts/build/macos/start-client.sh]: You need to add the flag `-XstartOnFirstThread` after `-jar`

File: scripts/build/macos/start-server.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cd ../../../bin/
+java -jar serverCM.jar
```
Comment (adellafrattina) [inline scripts/build/macos/start-server.sh]: You need to add the flag `-XstartOnFirstThread` after `-jar`

File: scripts/build/macos/build-client.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+cd ../../..
+rm -f bin/clientCM.jar
+rm -f bin/coreCM.jar
+./scripts/maven/mvnw -am --projects climate-monitoring-server clean install
```
Comment (adellafrattina) [inline scripts/build/macos/build-client.sh]: You area building for the server, but in this file you should build for the client

**Review by adellafrattina — CHANGES_REQUESTED** [review]
Good job. Just one question. Did you test these scripts? If not, try them out, because I need them to work perfectly

**Comment by francescolops** [comment]
explain me how can I test them and I'll do right now boss

**Review by adellafrattina — APPROVED** [review]
Should be good

**Pull request merged**